### PR TITLE
Add F-4E Phantom to the list of CAS/STRIKE capable planes

### PR DESCRIPTION
**Pull request by Hornet2041**
#1072 

 adds F-4E to list of CAS and STRIKE capable missions

### CODE REVIEW

File: gen/flights/ai_flight_planner_db.py [hunk 1]
```diff
@@ -174,6 +174,7 @@
     Su_25,
     F_15E,
     F_16C_50,
+    F_4E,
```
Comment (DanAlbert) [inline gen/flights/ai_flight_planner_db.py]: These are a priority order, so putting it here means that the F-4 will be preferred for CAS over the F/A-18. I don't think that was your intent. Probably makes more sense below the Harrier?

File: gen/flights/ai_flight_planner_db.py [hunk 2]
```diff
@@ -285,6 +286,7 @@
     AJS37,
     Tornado_GR4,
     F_16C_50,
+    F_4E,
```
Comment (DanAlbert) [inline gen/flights/ai_flight_planner_db.py]: Ditto.

**Review by DanAlbert — CHANGES_REQUESTED** [review]

**Comment by Hornet2041** [comment]
oh very cool, hadn't realized that was a priority order. I stuck them lower on the list since its an older aircraft and can only carry rockeyes, mk82/84/gbu-12/10 and rockets

**Review by DanAlbert — APPROVED** [review]
Looks good now, thanks!

**Pull request merged**